crates/api_models/src/payments.rs

```diff
@@ -3765,6 +3785,15 @@ pub struct Address {
     pub email: Option<Email>,
 }
 
+#[derive(Default, Clone, Debug, Eq, PartialEq, ToSchema, serde::Deserialize, serde::Serialize)]
+#[serde(deny_unknown_fields)]
+pub struct MerchantURLs {
```

Comment:
what is this used for? what values should be passed, is this specific to only one connector, if so please decleare this in the respective connector module